### ISSUE TITLE
[ODS-App] About > Accessibility-Statement

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ods-ios#119]
# Accessibility-Statement

## Requirement

**As a** user of the demo app
**I want** toview the Accessibility Statement
**So That** I check the conformity of the app to the a11y requirements

## General description
In the About page, a new "Accessibility Statement" list item will propose to open a new page.

The new page will present the result of the [Conformity audit WCAG / IOS](https://la-va11ydette.orange.com/?list=wcag-ios&lang=en). The output of the Accessibility Conformity Audit will be a json file (or xml). The Accessibility Statement Page will automatically display the content based on the json file.

Here is an example of a json file to use: [statementData (3).zip](https://github.com/Orange-OpenSource/ods-android/files/8799601/statementData.3.zip)
(or  [ods.zip](https://github.com/Orange-OpenSource/ods-android/files/8799605/ods.zip) )

- the file to use was added in task #120

## UI specifications

DESIGN TO BE CREATED

Here is an example of the mDAN App Conformity Statement:
<img src="https://user-images.githubusercontent.com/62606284/171027918-4caaac38-8f5f-40d0-8ed7-06156eef9826.png" width="300"/>

## Implementation Tasks
- [x] #120
- [ ] Accessibility-Statement Page Design
- [ ] Accessibility-Statement component Creation

**Comment by B3nz01d** [comment]
The table of the conformity, should be displayed to be compatible on mobile displays. The table should not display the non applicable column. The table cells should display the conformity information "9A - 5AA".

**Comment by B3nz01d** [comment]
Initial discussion identified to first evaluate the integration of the existing SDK, an then decide to perform quick integration, then later define an internal version of the sdk within the lib.

**Comment by Tayebsed93** [comment]
Ressource xml et html [accessibility.zip](https://github.com/Orange-OpenSource/ods-ios/files/8992036/accessibility.zip)

**Comment by Tayebsed93** [comment]
lib : [https://gitlab.tech.orange/tayeb.sedraia/accessibility_ios_accessibility_statement.git](https://gitlab.tech.orange/tayeb.sedraia/accessibility_ios_accessibility_statement.git)

**Comment by ludovic35** [comment]
Some remarks need to be aanswered:
- How to use the ODS theme and components (Buttons, Colors, ...) in DeclarationAccessibility package
- Is the accessibility result available only from the About Page ?

**Comment by B3nz01d** [comment]
- @mccart77  what button emphasis should be used in this page?
- [ ] @Tayebsed93  the button in the page should be using the ods
